packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -466,7 +466,7 @@ export interface FunctionOptions extends EventInvokeConfigOptions {
    * Sets the logFormat for the function.
    * @default Text format
    */
-  readonly logFormat?: string;
+  readonly logFormat?: LogFormat;
```

Comment:
I think this would be a breaking change since it's currently possible (arguably encouraged by the type) to set the property to the strings "JSON" or "Text".

I suggest adding `| keyof LogFormat` which would let the user use either the enum or type-safe strings

```suggestion
  readonly logFormat?: LogFormat | keyof typeof LogFormat;
```

Reply:
Thank you for your comment.
I will modify logFormat type to accept both the enum and type-safe strings

Reply:
Thank you for your suggestion.
Your comment realised me that my first change would be a breaking change.
Both of my first change and your suggestion can't accept both of the enum or type-safe strings.
Therefore, I changed the type of `logFormat` as ``readonly logFormat?: LogFormat | `${LogFormat}`;`` due to accept both of `lambda.LogFormat.TEXT` and string `'Text'`.